orchestrator/handlers/verification_handler.py

```diff
@@ -21,3 +21,13 @@ def get_fields_handler(event, context):
     country_code = event["pathParameters"]["countryCode"]
     response = Service.get_fields(configuration_name, country_code)
     return generate_lambda_response(StatusCode.OK, {"status": ResponseStatus.SUCCESS, "data": response})
+
+
+@handle_exception_with_slack_notification(logger=logger, SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID)
+def get_fields_handler(event, context):
+    required_keys = ["countryCode"]
+    if not validate_dict(event["pathParameters"], required_keys):
+        raise BadRequestException()
+    country_code = event["pathParameters"]["countryCode"]
+    response = Service.get_document_types(country_code)
```

Comment:
rename service to VerificationService

Reply:
VerificationService is the class. So instead of initiating it everytime, I'm initiating it once in the top and storing it in service

Reply:
https://github.com/singnet/snet-marketplace-service/pull/852/commits/c1fd65a530e4f2339133a33ca54b3b281b371f13